scripts/configure/zowe-configure.sh

```diff
@@ -59,9 +59,10 @@ echo "Beginning to configure zowe installed in ${ZOWE_ROOT_DIR}"
 
 # Configure API Mediation layer.  Because this script may fail because of priviledge issues with the user ID
 # this script is run after all the folders have been created and paxes expanded above
-echo "Attempting to setup Zowe API Mediation Layer certificates ... "
-. $CONFIG_DIR/zowe-configure-api-mediation.sh
+#echo "Attempting to setup Zowe API Mediation Layer certificates ... "
+#. $CONFIG_DIR/zowe-configure-api-mediation.sh
```

Comment:
I think this change should be undone and the above changes made the the zowe-configure-api-mediation.sh to try and keep backwards compatibility working short term